docs/platforms.md

```diff
@@ -0,0 +1,41 @@
+# Platforms
+
+This document outlines the supported platforms for matrix.org bridge projects.
+
+### Bridge versioning
+
+All bridge application and library projects aim to follow the [Semantic Versioning](https://semver.org/) system.
+
+### Node version support
+
+Bridge projects will always support the **Active LTS** release and the **Current** release of Node.JS. When a
+Node.JS version falls from Active to Maintenance, we will migrate projects to the next Active release over time.
```

Comment:
Debian is very hard to support because the stable versions pack out-of-LTS releases. We're never going to be able to support that. 

I'd *like* to support maintenance versions a bit longer, but the trouble is upstream projects like bot-sdk make that a bit difficult (it has a hard dep on Node 16+).